cv_next/app/profile/[profileId]/components/profileData.tsx

```diff
@@ -27,7 +27,7 @@ export default async function ProfileData({ user }: { user: UserModel }) {
     <div className="flex flex-col items-center">
       <div
         style={{ width: "100%" }}
-        className={`fill-available mb-3 rounded-lg border-b border-gray-200 bg-white p-6 text-base dark:bg-theme-800`}
+        className={`fill-available mb-3 rounded-lg border-b border-card-border p-6 text-base bg-card-background`}
```

Comment:
It seems like this card style is used in multiple places.
Perhaps create a UI component for `<Card/>`?

Reply:
Also if you see other similar generic components-styles, we should turn them into proper components.

Reply:
i dont think we need to be this generic, card can take multiple forms and not be the same anywhere.
cards can be comments, sections, proper card and more